docs/excel/excel-data-types-concepts.md

```diff
@@ -58,23 +60,33 @@ The following sections show JSON code samples for the formatted number value, en
 
 ## Formatted number values
 
-The [FormattedNumberCellValue](/javascript/api/excel/excel.formattednumbercellvalue) object enables Excel add-ins to define a `numberFormat` property for a value. Once assigned, this number format travels through calculations with the value and can be returned by functions.
+The [DoubleCellValue](/javascript/api/excel/excel.doublecellvalue) object enables Excel add-ins to define a `numberFormat` property for a value. Once assigned, this number format travels through calculations with the value and can be returned by functions.
 
```

Comment:
Should we add a note here that FormattedNumberCellValue has been deprecated? Maybe something like this? 
```suggestion

> [!IMPORTANT]
> [FormattedNumberCellValue](/javascript/api/excel/excel.formattednumbercellvalue) has been deprecated. Use [DoubleCellValue](/javascript/api/excel/excel.doublecellvalue) to define number format properties.

```

Reply:
Seems like we should just update our docs to always use DoubleCellValue. The reference docs should indicate this is deprecated. Yes?

Reply:
Ok, that works for me